components/query_handler/queryHandler.py

```
@@ -1,33 +1,65 @@
     async for message in websocket:
-        print(f"Received message: {message}")
-
-        message = json.loads(message)
+        message = json.loads(message)        
 
+        print(f"Received message: {getMessageFromCode(message['code'])}")
+        
         if "code" not in message:
-            message["response"] = "Message does not contain a code"
-            message["error"] = True
+            handleUnknownMessage(
```

Ruff (0.8.2)

`handleUnknownMessage` may be undefined, or defined from star imports

(F405)